# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/msgpuck/cmake4.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

CMake 4 no longer allows the OLD CMP0037 behavior, rename the reserved
"test" custom target to "check" instead.

--- msgpuck-2.0.orig/test/CMakeLists.txt	2017-02-07 13:14:17.000000000 +0100
+++ msgpuck-2.0/test/CMakeLists.txt	2026-09-25 19:43:42.314570173 +0200
@@ -1,7 +1,3 @@
-if(POLICY CMP0037)
-    cmake_policy(SET CMP0037 OLD) # don't blame custom target names
-endif(POLICY CMP0037)
-
 include_directories("../")
 
 #find_program(PROVE prove)
@@ -22,4 +18,4 @@ foreach (test msgpuck)
         COMMAND ${TEST_RUNNER} ${PROJECT_BINARY_DIR}/test/${test}.test)
 endforeach()
 
-add_custom_target(test DEPENDS ${alltests})
+add_custom_target(check DEPENDS ${alltests})
